server/superdesk/io/rss.py

```diff
@@ -307,6 +310,9 @@ def _create_package(self, text_item, image_items):
         """
         package = {
             'type': 'composite',
+            'guid': 'package:' + text_item['guid'],
```

Comment:
BTW, what to use for the "hint" to the linked function, simply the GUID_TAG from the settings?